tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -0,0 +1,1497 @@
+// TODO(jwexler): Do we need these deps?
```

Comment:
We shouldn't, if they get schlepped in transitively via the JS build graph.

Please note, we also have this at the moment:

- https://github.com/tensorflow/tensorboard/blob/master/tensorboard/components/tf_imports/BUILD#L8
- https://github.com/tensorflow/tensorboard/blob/master/tensorboard/components/tf_imports/closure.html

Which shouldn't apply to you. It's meant to keep raw source tensorboard builds tiny, so we don't have to wait on closure compiler.

Reply:
done